classic_startup/python2.7libs/pythonrc.py

```diff
@@ -43,6 +44,26 @@ def classic_startup():
     # add to the system path
     sys.path.insert(0, tk_houdini_python_path)
 
+    # We need to check to make sure we don't have an incompatibility
+    # between httplib and Houdini's bundled ssl.py. This is a problem
+    # on some Linux distros (CentOS 7.x) with H16.
+    if sys.platform == "linux2" and sys.version.startswith("2.7.5"):
```

Comment:
Recommend 
```python
if sys.platform.startswith("linux") and ...
```
as per: https://docs.python.org/2.7/library/sys.html#sys.platform for eventual compatibility with python `3.3+`.